force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -12077,6 +12077,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>oppNamingOrganizational</shortDescription>
         <value>Organization Donations</value>
     </labels>
+    <labels>
```

Comment:
@ErinWiedemer Can you review the following error message for payment/refund.

Reply:
@andrewyu-salesforce I spoke with @melissabarber about this warning message because I had not seen it before. It hasn't been UI Text reviewed either. She told me that she thinks there's an existing message that you could reuse that says, "You can't change {field label} for an Elevate-connected Recurring Donation." We looked for it, but we couldn't find it. If you can find that, she said that it would be best to edit it to say, "You can't change {field label} for an Elevate-connected Recurring Donation because the field is important for the Elevate integration." Will that work?

Reply:
@andrewyu-salesforce also you are missing an "e" in error on line 12101 12109.

Reply:
The label is copied from the RD validation. The original label is 'Recurring Donations from Elevate Payment Services can&apos;t be modified in NPSP except by the Elevate Payment Services Integration user.`

Reply:
@andrewyu-salesforce do you know where the error highlighted in this screenshot comes from and could we use that label in this context or do we need a slightly different one?
<img width="830" alt="Screen Shot 2022-06-14 at 11 38 56 AM" src="https://user-images.githubusercontent.com/19452076/176315610-1daa65b9-0172-4322-871f-421038d38eae.png">

Reply:
@melissabarber The labels you are referencing are used for editing Elevate field that is not supported for edit. In this case, we are entirely blocking any modification for a group of fields for the non-integration user. A similar label is also used in RD2. It's the `RD2_ElevatePermissionRequired` label.

Reply:
The fields we are blocking are:
- npe01__Payment_Amount__c
- Elevate_Payment_ID__c
- npe01__Payment_Date__c
- DebitType__c
- OriginalPayment__c

Reply:
@andrewyu-salesforce it looks like there are 4 custom labels here that haven't gone through the UI text review process. Do you have any design docs, mocks, or a demo that would show us when, where and how these error message occur? I could meet you for a walkthrough today, if that would be the fastest way to gain insight. I will take these messages to review this afternoon, but we really need some context to format them correctly.